data/blog/bitcoin-infrastructure-this-decade.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Looking Ahead: Bitcoin Infrastructure This Decade"
```

Comment:
Perhaps better to use "Bitcoin's Infrastructure" versus "Bitcoin Infrastructure?"